test/repository_test.rb

```diff
@@ -18,18 +19,25 @@
       before do
         UserRepository.adapter    = adapter.new(mapper, uri)
         ArticleRepository.adapter = adapter.new(mapper, uri)
-
+        
         UserRepository.collection    = :users
         ArticleRepository.collection = :articles
 
         UserRepository.clear
         ArticleRepository.clear
+
+        unless adapter_name == :file_system
```

Comment:
Because the `FileSystem::Adapter` require define Phone collection. If not, the test will encounter bug:
```
Lotus::Model::Adapters::FileSystemAdapter::#info#test_0001_returns infos per each collection:
Lotus::Model::Mapping::UnmappedCollectionError: Cannot find collection: phones
```